decentraland/unity-renderer
# fix: biw smart item compatibility

Fixes #744 
## What does this PR change?

This PR adds the compatibility with builder around smart items so you can load the builder scenes even if they have smart items inside it 

It also include a quick fix

- [x] If you try to enter in builder in world while the scene is still not loaded (from the shorcut) it won't start the builder in world flow 

## How to test the changes?

1. Go to: https://play.decentraland.zone/branch/feat/biw-smart-item-handle/index.html?renderer=urn:decentraland:off-chain:renderer-artifacts:fix/biw-fix-bug
2. Open builder in world land with an scene that you has previously created in builder with smart items 
3. Made a change in the scene and open again the scene in the builder
4. All the connections in the smart items should remain

## Our Code Review Standards

https://github.com/decentraland/unity-renderer/blob/master/docs/code-review-standards.md
